src/diffusers/pipelines/onnx_utils.py

```diff
@@ -172,7 +172,7 @@ def _from_pretrained(
         # load model from local directory
         if os.path.isdir(model_id):
             model = OnnxRuntimeModel.load_model(
-                os.path.join(model_id, model_file_name), provider=provider, sess_options=sess_options
+                Path(model_id, model_file_name).as_posix(), provider=provider, sess_options=sess_options
```

Comment:
Don't we need to import `Path`?

Reply:
Yes, but it's already imported because Path is also being used in other parts of the file